internal/background/db_stats.go

```diff
@@ -18,15 +18,13 @@ func dbStatsLoop(ctx context.Context, sleep time.Duration) {
 	}()
 	ticker := time.NewTicker(sleep)
 
+	// run one tick immediately to prevent prometheus gaps
+	dbStatsObserveTick(ctx)
```

Comment:
Is That what you meant by there is no other way for the go builds in tick to start immediately?

Reply:
Yeah I need to trigger it manually. There is no function in the standard library to do it AFAIK: https://pkg.go.dev/time#NewTicker